js/adapt-offlineStorage-xapi.js

```diff
@@ -58,11 +58,12 @@ define([
     },
 
     /**
-     * @returns {{name: string in the format Firstname Lastname, firstname: string, lastname: string }}
+     * @returns {{id: string, name: string in the format Firstname Lastname, firstname: string, lastname: string }}
```

Comment:
```suggestion
     * @returns {{id: string, name: string, firstname: string, lastname: string}} The learner's id (Actor.account.name), full name (in the format Firstname Lastname), first and last names
```

In VSC, that generates the following:
![2019-02-14 11_37_29-adapt-offlinestorage-xapi js - dev - visual studio code](https://user-images.githubusercontent.com/1249597/52784658-c2c7cd80-304d-11e9-9ba3-9cf2a96fb7ea.png)